applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py

```diff
@@ -74,6 +77,14 @@ def AssambleTestSuites():
     nightSuite.addTest(TimeIntegratedFluidElementTest('testSymbolic'))
     nightSuite.addTest(FluidAnalysisTest('testFluidDynamicsAnalysis'))
     nightSuite.addTest(AdjointFluidTest('testCylinder'))
+    nightSuite.addTest(AdjointVMSElement2D('testCalculateSecondDerivativesLHS'))
+    nightSuite.addTest(AdjointVMSElement2D('testCalculateFirstDerivativesLHS1'))
+    nightSuite.addTest(AdjointVMSElement2D('testCalculateFirstDerivativesLHS2'))
+    nightSuite.addTest(AdjointVMSElement2D('testCalculateSensitivityMatrix'))
+    nightSuite.addTest(AdjointVMSSensitivity2D('testCylinder'))
+    nightSuite.addTest(AdjointVMSSensitivity2D('testOneElement'))
+    nightSuite.addTest(AdjointCPPUnitTests('testSensitivityCPP'))
```

Comment:
Here the cpp unit tests can be called like in https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py#L369.

Reply:
Done :)